### DISCUSSION TITLE
preload is found, but is not used because the script type does not match.

### DISCUSSION THREAD

**Discussion by FranGomezVenegas** [open-wc/open-wc#2210] [Q&A]
Hi All,

I just created one new project as type application and with no type-script, when I run it by run start in local it works. I can create the build but then when I publish it in my webserver then it returns this error with a reference to the .js file, "preload is found, but is not used because the script type does not match."

In the index.html the reference is of this manner:  <script type="module" src="...

Any idea of how to unlock this situation? 

Thanks in advance

**Comment by Westbrook** [comment]
Chrome made a change here that they are reverting to cause this, so it should just go away soon. However, we’re looking at converting this to `modulepreload` everywhere internally, so this no longer occurs regardless. We’ll have some new rewards around this, soon.

**Reply by FranGomezVenegas** [reply]
Ok, Thanks for reply. I will wait to this be fixed by Chrome then. My concern was on having this stoppable only by building the base project .... 
Thanks again

**Reply by FranGomezVenegas** [reply]
Hi @Westbrook , only one last question, do you know in which issue I can track this topic? I mean, to know when it will be fixed.
Thanks
